packages/reactor-browser/src/hooks/drive-context.tsx

```
@@ -24,12 +41,10 @@ export const DriveContextProvider: React.FC<
  *   addFile
  * } = useDriveContext();
  */
-export function useDriveContext() {
-  const context = useContext(DriveContext);
-  if (!context) {
-    throw new Error(
-      "useDriveContext must be used within a DriveContextProvider",
-    );
-  }
-  return context;
+export function useDriveContext(): IDriveContext {
+  const driveContext = useSyncExternalStore(
+    subscribeToDriveContext,
+    () => window.phDriveContext,
+  );
+  return driveContext ?? DEFAULT_DRIVE_CONTEXT;
```

I'd just throw an error if `driveContext` is undefined. The default context will just hide potential errors in the setup.